BTCPayServer/Security/GreenField/APIKeyRepository.cs

```diff
@@ -52,6 +55,26 @@ public async Task CreateKey(APIKeyData key)
             await context.SaveChangesAsync();
         }
 
+        public async Task UpdateKey(string id, string[] permissions, string label, string userId)
```

Comment:
Avoid strings, prefer proper types
```suggestion
        public async Task UpdateKey(string id, Permission[] permissions, string label, string userId)
```